Find a functionally equivalent code:android.text.BoringLayout.init:COMMENT
Method Modifier: 
Comment:/* package */

@@ -11,19 +11,16 @@
     } else {
         spacing = metrics.descent - metrics.ascent;
     }
-    if (spacingmult != 1 || spacingadd != 0) {
-        spacing = (int) (spacing * spacingmult + spacingadd + 0.5f);
-    }
     mBottom = spacing;
     if (includepad) {
         mDesc = spacing + metrics.top;

